src/Components/LoginSignUp/SignUpForm.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useRef } from 'react'
+
+const SignUpForm = (props) => {
+    var newSignUp = {}
```

Comment:
You can use Form in this file instead of handling all the functions independently.

Reply:
Fixed